js/components/portal-dashboard/feedback/activity-feedback-textarea.tsx

```diff
@@ -29,23 +30,30 @@ export const ActivityFeedbackTextarea: React.FC<IProps> = (props) => {
 
   const updateFeedback = () => {
     if (activityId && studentId && updateActivityFeedback && textareaRef.current?.value !== undefined) {
+      const hasBeenReviewed = textareaRef.current.value !== "" ? true : false;
```

Comment:
I was wondering about this case where the user deletes the feedback.  Good catch!